doc/2/guides/advanced/debugging/index.md

```diff
@@ -0,0 +1,108 @@
+:::
+
+Once the proxy server has started and has successfuly connect to your Kuzzle instance will see a message like this in your terminal:
+```bash
+[ℹ] Connecting to ws://<host>:<port> ...
+[ℹ] Connected to Kuzzle node: knode-lackadaisical-archimedes-79903
+[✔] Listening on port 9222, forwarding to Kuzzle at <host>:<port>
+[ℹ] Showing to Chrome Debugger as "Kuzzle Debugger - <host>:<port>"
+[ℹ] Waiting for Chrome Debugger to connect...
+```
+
+```bash
+[✔] Listening on port 9222, forwarding to Kuzzle at <host>:<port>
+```
+You can see here that the proxy server is listening on port `9222` this will be needed for the next step.
+
+#### Step 2 - Configure Chrome Inspector
+
+The proxy server is now setuped, you now need to open the Chrome Inspector to debug your instance, to do that you can open this url `chrome://inspect` in [Chrome Browser](https://www.google.com/intl/en_en/chrome/).
+This will open you the debug interface of Chrome which will then be used to debug the Kuzzle instance.
+
+To allow the Chrome Inspector to see our proxy server exposed by the command `kourou app:debug-proxy`
+we need to add `localhost:9222` to the list of targets the Chrome Inspector can see and debug.
+
+:::info
+Here we are using the port `9222` but keep in mind that this port can be any other number and you need to change it accordingly on what port the proxy server is listening to, see end of [Step 1 - Setup proxy server](#step-1---setup-server-proxy)
+
+The host will always be localhost since it's the IP of the proxy server that is running on your local machine.
+:::
+
+**1 -** Click on the `Configure` button, a dialog will appear where you can enter the IP and Port of your local proxy server.
+![Image Configure Chrome Inspector Step 1](./config_step_1.png)
+
+**2 -** Enter your local target, here  `localhost:9222`
+**3 -** Confirm
+![Image Configure Chrome Inspector Step 2](./config_step_2.png)
+
+**If everything worked so far you should see a new Remote Target called** `Kuzzle Debugger - <Instance Host>:<Instance Port>`.
+
+![Image Configure Chrome Inspector Step 3](./config_step_3.png)
+
+**4 -** You can now click on `inspect` and your done, you can now debug your instance remotely using the Chrome Inspector.
+
+#### Step 3 - Debugging
+
+You can now fully use the Chrome Inspector to debug you instance.
+
+#### Step 4 - After Debugging
+
+:::warn
+Don't forget to set the config `security.debug.native_debug_protocol` back to `false` once your done debugging to increase the security of your instance in case of one of your admin users with access to the [Debug Controller](/core/2/api/controllers/debug) gets compromised.
```

Comment:
```suggestion
Don't forget to set the `security.debug.native_debug_protocol` configuration setting back to `false` once you're done debugging to increase the security of your instance in case of one of your admin users with access to the [Debug Controller](/core/2/api/controllers/debug) gets compromised.
```